src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs

```diff
@@ -14,7 +14,7 @@ internal partial class FeatureFlag : IFeatureFlag
 
     private const string Prefix = "VSTEST_FEATURE_";
```

Comment:
```suggestion
    private const string VSTEST_FEATURE = nameof(VSTEST_FEATURE);
```

Reply:
And do the same for the rest of the env variables. And join them with `+ "_" +`.

Also the comment says "po*r*st-processing"